app/models/quiz_response.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class QuizResponse < ApplicationRecord
+  include Updateable
+
+  belongs_to :quiz_question
+  has_and_belongs_to_many :mindsets, inverse_of: nil
+
+  def self.update_params
+    params = super
+    params << { mindset_ids: [] }
```

Comment:
`has_and_belongs_to_many` relations don't appear in the hash returned by the `attributes` method, so I'm extending this method for that case.

Reply:
Is the goal of these methods to do a blanket `permit` on all incoming attributes? Do we have any cases where we plan to restrict attributes?

Reply:
Yes, the goal is to be able to limit the parameters to the actual fields on the model. I just want to prevent any potential problems that users of the API can encounter by sending along random information 😅 There are currently no cases where we need to restrict the attributes.